tests/test_monitoring.py

```diff
@@ -835,7 +835,14 @@ def test_send_immediate_email_alerts(client, app):
         'products': [{'code': '12345'}],
         "versioncreated": utcnow(),
         'byline': 'Testy McTestface',
-        'body_html': '<p>line 1 of the article text\nline 2 of the story\nand a bit more.</p>',
+        'body_html': '<p>line 1 of the article text\nline 2 of the story\nand a bit more.</p>'
+                     '<!-- EMBED START Audio {id: "editor_2\"} -->'
+                     '<figure>'
+                     '    <audio controls src="/assets.mp3"</audio>'
```

Comment:
Looks like the `audio` tag is not closed, missing closing bracket in tag open

Reply:
Fixed